Makefile.toml

```diff
@@ -704,6 +704,7 @@ cargo install \
   --jobs ${BUILDSYS_JOBS} \
   --root tools \
   --quiet \
+  --version 0.7.1 \
```

Comment:
Should this be a variable in the Makefile? I worry we'll forget that this is buried in here.